esphome/components/uart/event/__init__.py

```diff
@@ -0,0 +1,76 @@
+import esphome.codegen as cg
+from esphome.components import event, uart
+import esphome.config_validation as cv
+from esphome.const import CONF_EVENT_TYPES
+
+DEPENDENCIES = ["uart"]
+
+uart_ns = cg.esphome_ns.namespace("esphome").namespace("uart")
```

Comment:
This creates esphome::esphome::uart

```suggestion
uart_ns = cg.esphome_ns.namespace("uart")
```

Reply:
Could just be `from .. import uart_ns`